setup.py

```diff
@@ -148,8 +148,8 @@ def customize_compiler(compiler):
             if platform.system() == 'Darwin':
                 flags.append('-mmacosx-version-min=10.7',)
                 if tuple(
-                   map(int, platform.mac_ver()[0].split('.'))
-                   ) >= (10, 9):
+                map(int, platform.mac_ver()[0].split('.'))
```

Comment:
this should be indented 8 more spaces

Reply:
Why is that?

Reply:
https://www.python.org/dev/peps/pep-0008/#indentation

notably the part where it says:
"More indentation included to distinguish this from the rest."